cmd/zoekt-sourcegraph-indexserver/cleanup.go

```diff
@@ -74,14 +74,23 @@ func cleanup(indexDir string, repos []string, now time.Time) {
 		shardsLog(indexDir, "restore", shards)
 	}
 
-	// index: Move non-existant repos into trash
-	for _, shards := range index {
+	// index: Move non-existent repos into trash
+	for repo, shards := range index {
 		// Best-effort touch. If touch fails, we will just remove from the
 		// trash sooner.
 		for _, shard := range shards {
 			_ = os.Chtimes(shard.Path, now, now)
 		}
 
+		if _, err := os.Stat(filepath.Join(indexDir, zoekt.TombstoneFileName)); err == nil {
+			if len(shards) > 0 && strings.HasPrefix(filepath.Base(shards[0].Path), "compound-") {
+				shardsLog(indexDir, fmt.Sprintf("setTombstone %s", repo), shards)
```

Comment:
this is breaking the awkness of shardsLog. Can we maybe add another column for repo name or id. IE a third optional column for repo name.